src/sagas/templatesSaga.js

```diff
@@ -33,7 +36,21 @@ export function* addNewTemplateToStore() {
   yield put(actions.addNewTemplateSuccess(newTemplate));
 }
 
+/**
+ * saga which checks if template is in the store
+ * and loads the template if it is not
+ */
+export function* addTemplateObjectToStore(action) {
```

Comment:
This could have a try/catch and thus an error action associated with it.

Reply:
good call. will add, thanks!

Reply:
Would this be integrated with the `Error` container in the future?

Reply:
I'm not sure if it makes sense to surface this type of error in the error logger (which is more for errors from a user editing a template to help them edit it correctly). I think the UI will dictate where these type of errors should be (maybe in a separate message-like component?).